common/Scripting/StoryboardObjectGenerator.cs

```diff
@@ -161,18 +161,19 @@ public float[] GetFft(double time, string path = null)
         /// Returns the Fast Fourier Transform of the song at a certain time, with the specified amount of magnitudes.
         /// Useful to make spectrum effets.
         /// </summary>
-        public float[] GetFft(double time, int magnitudes, string path = null, OsbEasing easing = OsbEasing.None)
+        public float[] GetFft(double time, int magnitudes, string path = null, OsbEasing easing = OsbEasing.None, int FrequencyCutOff = 20000, int SamplingRate = 44100)
```

Comment:
If possible, default values shouldn't change how existing storyboard look. 

Change in Spectrum and RadialSpectrum are fine because only newly created effects will have them.